<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(45, 45, 45);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: rgb(35, 35, 35);
	border-radius:5px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../resource/asset.qrc">:/icons/asset/settings.svg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color: rgb(255, 255, 255);
}</string>
              </property>
              <property name="text">
               <string>Settings</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_minimize">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: rgb(85, 255, 127);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(85, 255, 127,150);
	
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_maximize">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	
	background-color: rgb(255, 170, 0);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(255, 170, 0,150);
	
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_close">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: rgb(255, 0, 0);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(255, 0, 0,150);
	
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLineEdit" name="details_url">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>100</y>
              <width>470</width>
              <height>51</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>470</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>51</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="tabletTracking">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(255, 255, 255);
	border: 2px solid  rgb(45, 45, 45);
	border-radius:10px;
	padding-left: 45px;
}

QLineEdit:hover{
	border:2px solid rgb(35, 35, 35);
}

QLineEdit:focus{
	border:2px solid rgb(255, 255, 255);
}</string>
            </property>
            <property name="placeholderText">
             <string>Details url</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_36">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>110</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(45, 45, 45);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resource/asset.qrc">:/icons/asset/user.svg</pixmap>
            </property>
           </widget>
           <widget class="QLabel" name="label_notification">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>260</y>
              <width>470</width>
              <height>91</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>470</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>470</width>
              <height>101</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
}</string>
            </property>
            <property name="text">
             <string>Notification</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="current">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>40</y>
              <width>150</width>
              <height>20</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton{
	color: rgb(255, 255, 255);
	background-color:rgb(35,35,35)
}</string>
            </property>
            <property name="text">
             <string>Current College</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBox">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>30</y>
              <width>180</width>
              <height>41</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox{
	color: rgb(255, 255, 255);
	border: 2px solid rgb(45, 45, 45);
	padding-left:10px;
	border-radius: 5px;
}</string>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_update_properties">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>180</y>
              <width>221</width>
              <height>51</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>221</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>221</width>
              <height>51</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border:none;
	border-radius: 10px;
}

QPushButton:hover{
	border-radius: 20px;
}

QPushButton:pressed{
	border: 2px solid rgb(255,255,255);	
}</string>
            </property>
            <property name="text">
             <string>Update</string>
            </property>
            <property name="icon">
             <iconset resource="../resource/asset.qrc">
              <normaloff>:/icons/asset/repeat.svg</normaloff>:/icons/asset/repeat.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLineEdit" name="type">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>180</y>
              <width>221</width>
              <height>51</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>441</width>
              <height>51</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="tabletTracking">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(255, 255, 255);
	border: 2px solid  rgb(45, 45, 45);
	border-radius:10px;
	padding-left: 45px;
}

QLineEdit:hover{
	border:2px solid rgb(35, 35, 35);
}

QLineEdit:focus{
	border:2px solid rgb(255, 255, 255);
}</string>
            </property>
            <property name="placeholderText">
             <string>Type</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_39">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>190</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(45, 45, 45);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resource/asset.qrc">:/icons/asset/external-link.svg</pixmap>
            </property>
           </widget>
           <widget class="QRadioButton" name="colleges">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>40</y>
              <width>90</width>
              <height>20</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton{
	color: rgb(255, 255, 255);
	background-color:rgb(35,35,35)
}</string>
            </property>
            <property name="text">
             <string>Colleges</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/asset.qrc"/>
 </resources>
 <connections/>
</ui>
